mathics/core/symbols.py

```diff
@@ -754,7 +761,7 @@ def symbol_set(*symbols: Symbol) -> FrozenSet[Symbol]:
 
 def sympy_name(mathics_symbol: Symbol):
     """Convert a mathics symbol name into a sympy symbol name"""
-    return SYMPY_SYMBOL_PREFIX + mathics_symbol.name
+    return SYMPY_SYMBOL_PREFIX + mathics_symbol.name.replace("`", "_")
```

Comment:
`SYMPY_SYMBOL_PREFIX + mathics.symbol.name.replace("``'", "_")` seems like an idiom. Maybe it could be turned into a function?